dev-libs/pigpio/files/pigpio-66-makefile.patch

```diff
@@ -0,0 +1,72 @@
+--- a/Makefile
++++ b/Makefile
+@@ -12,24 +12,24 @@
+ 
+ CFLAGS	+= -O3 -Wall -pthread
+ 
+-LIB1     = libpigpio.so
++LIB1     = libpigpio.so.0
+ OBJ1     = pigpio.o command.o
+ 
+-LIB2     = libpigpiod_if.so
++LIB2     = libpigpiod_if.so.0
+ OBJ2     = pigpiod_if.o command.o
+ 
+-LIB3     = libpigpiod_if2.so
++LIB3     = libpigpiod_if2.so.0
+ OBJ3     = pigpiod_if2.o command.o
+ 
+ LIB      = $(LIB1) $(LIB2) $(LIB3)
+ 
+ ALL     = $(LIB) x_pigpio x_pigpiod_if x_pigpiod_if2 pig2vcd pigpiod pigs
+ 
+-LL1      = -L. -lpigpio -pthread -lrt
++LL1      = -L. $(LIB1) -pthread -lrt
+ 
+-LL2      = -L. -lpigpiod_if -pthread -lrt
++LL2      = -L. $(LIB2) -pthread -lrt
+ 
+-LL3      = -L. -lpigpiod_if2 -pthread -lrt
++LL3      = -L. $(LIB3) -pthread -lrt
+ 
+ prefix = /usr/local
```

Comment:
I did not look into its internals, but it seems you should fix the prefix path here too, otherwise it is going to be installed into `/usr/local` which is not ok

Reply:
seems to work? I'm installing them with dobin and dolib.so etc.. and that doesn't have any problems... the files finally installed are not in /usr/local but are correctly put in /usr/bin etc..
Not sure if it makes a difference where the files are installed during the compile/prepare?

Reply:
I just try to understand why did you try to patch the Makefile and then avoided it installing all the things manually.

Reply:
The makefile patch is mostly just to have .so library versioning and also to not strip the files but instead let emerge do that (basically I just fixed the complaints that emerge had :-)
Don't know if it makes a different if i change /usr/local to /usr, i think the other changes would still be necessary?
I could maybe have a look tomorrow to see if it might simplify a few things to just change the prefix..

Reply:
I would recommend you to just update the patch (if you have already started working on it), rather than install things manually here, as for strip, it should not be difficult to drop the strip option fom the Makefile using updated patch file :)

Reply:
Ok, after fixing the mandir as well in the Makefile and adding the links there the only thing remaining in the ebuild are the systemd and openrc files.. hope that's better :-)